uaclient/entitlements/fips.py

```diff
@@ -163,9 +166,16 @@ def application_status(self) -> "Tuple[status.ApplicationStatus, str]":
         super_status, super_msg = super().application_status()
         if super_status != status.ApplicationStatus.ENABLED:
             return super_status, super_msg
-        running_kernel = util.get_platform_info()["kernel"]
-        if running_kernel.endswith("-fips"):
-            return super_status, super_msg
+
+        if os.path.exists(self.FIPS_PROC_FILE):
+            if util.load_file(self.FIPS_PROC_FILE) == 1:
```

Comment:
load_file doesn't return an int, so this will fail to ever show ua fips status as enabled I think
```bash
python3 -c 'from uaclient.util  import load_file; print(type(load_file("/proc/sys/crypto/fips_enabled")))
<class 'str'>

```